utils/htmlToPDF.ts

```diff
@@ -0,0 +1,32 @@
+import { create, CreateOptions } from "html-pdf";
+import { readFileSync } from "fs";
+
+/**
+ * converts hard coded html into a PDF and downloads the PDF locally
+ * @param {string} downloadPath download path of the generated pdf
+ * @param {string} volunteerName volunteer name
+ * @param {string} date date that the form will be filled out on, can we just pass in current date?
+ * @param {string} coordinatorName program coordinator name
+ * TODO: incorporate parameters into pdf after form html is provided
+ */
+const htmlToPDF = (
```

Comment:
how is this function going to be used? i don't think this should be doing any downloading but rather returning a PDF from the function - is that possible? the caller can do whatever they want with the file (store it, download it, etc.)

Reply:
I think this function will be used in the frontend for generating/downloading the PDF